x-pack/plugins/apm/public/components/shared/links/apm/transaction_detail_link/index.tsx

```diff
@@ -20,6 +19,7 @@ import { getComparisonEnabled } from '../../../time_comparison/get_comparison_en
 import { TruncateWithTooltip } from '../../../truncate_with_tooltip';
 import { APMQueryParams } from '../../url_helpers';
 import { APMLinkExtendProps, getLegacyApmHref } from '../apm_link';
+import { MaxGroupsMessage } from '../service_link/max_groups_message';
```

Comment:
nit: what about moving the component to the shared folder since it's shared between services and transaction link?

Reply:
I will move it outside the `servic_link` folder. 👍🏼